packages/ChangesReloaded-Core.package/CRBrowser.class/instance/switchToDetailedView.st

```diff
@@ -1,5 +1,5 @@
 button-actions
 switchToDetailedView
 	
-	CHBrowser open.
-	self changed: #close
\ No newline at end of file
+	self changed: #close.
+	^ CHBrowser open
```

Comment:
So rum geht das? Spannend. Warum ist das besser als das andere?

Reply:
Wir brauchen in den Tests eine Möglichkeit an die actual Fenster der Browser zu kommen, deswegen returnen wir die hier einfach. Analog auch bei switchToSimpleView. Und warum sollte das nicht gehen? Closen heißt afaik ja nicht deleten, das macht dann der garbage collector und der ist halt smart und collected nicht, bevor wir coole Dinge gemacht haben.

Reply:
Okay, sweet :)